### PR TITLE
SHIELD_CVEs_Jan_2022_2 (CVE-2021-1106 thru CVE-2021-1108)

### PR DESCRIPTION
**Pull request by PSIRT-NVIDIA**
SHIELD_CVEs_Jan_2022_2 (CVE-2021-1106 thru CVE-2021-2021-1108)

### CODE REVIEW

File: 2021/1xxx/CVE-2021-1106.json [hunk 1]
```diff
@@ -11,11 +11,21 @@
                     "product": {
```
Comment (cve-team) [inline 2021/1xxx/CVE-2021-1106.json:11]: 'version_value' is a required property

Comment (cve-team) [inline 2021/1xxx/CVE-2021-1106.json:11]: 'version_value' is a required property

File: 2021/1xxx/CVE-2021-1107.json [hunk 2]
```diff
@@ -11,11 +11,21 @@
                     "product": {
```
Comment (cve-team) [inline 2021/1xxx/CVE-2021-1107.json:11]: 'version_value' is a required property

Comment (cve-team) [inline 2021/1xxx/CVE-2021-1107.json:11]: 'version_value' is a required property

File: 2021/1xxx/CVE-2021-1108.json [hunk 3]
```diff
@@ -10,12 +10,22 @@
                 {
```
Comment (cve-team) [inline 2021/1xxx/CVE-2021-1108.json:10]: 'version_value' is a required property

Comment (cve-team) [inline 2021/1xxx/CVE-2021-1108.json:10]: 'version_value' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by PSIRT-NVIDIA** [comment]
re-submitted, the file with changes

**Comment by PSIRT-NVIDIA** [comment]
Can you provide the status for the re-submitted files submission?

**Comment by anthnysingleton** [comment]
CVE Records have been pulled in and Processed. This PR will be merged shortly.

**Comment by PSIRT-NVIDIA** [comment]
Thanks for the update! 

Milind

**Pull request merged**